hashMap.py

```diff
@@ -0,0 +1,65 @@
+
+# Time Complexity : O(n) for put, get and remove
+# Space Complexity : O(n)
+# Did this code successfully run on Leetcode : yes
+# Any problem you faced while coding this : Confused regarding the initialization of nodes array .
```

Comment:
https://stackoverflow.com/questions/22225666/pre-allocating-a-list-of-none This might help. Revert if you're still confused.